log4j-1.2-api/src/test/java/org/apache/log4j/config/AbstractLog4j1ConfigurationTest.java

```diff
@@ -276,7 +276,7 @@ private void testDailyRollingFileAppender(final String configResource, final Str
 
     public void testConsoleEnhancedPatternLayout() throws Exception {
         final PatternLayout layout = (PatternLayout) testConsole("config-1.2/log4j-console-EnhancedPatternLayout");
-        assertEquals("%d{ISO8601} [%t][%c] %-5p %properties %ndc: %m%n", layout.getConversionPattern());
+        assertEquals("%d{ISO8601} [%t][%c] %-5v1Level %properties %ndc: %m%n", layout.getConversionPattern());
```

Comment:
I think we need a comment before this line and one like it because I am guessing that "v1Level" was converted by us when users say "level" in their 1.2 configuration files?